cpu/atmega_common/periph/spi.c

```diff
@@ -56,6 +56,9 @@ void spi_init_pins(spi_t bus)
 #ifdef CPU_ATMEGA328P
     DDRB |= ((1 << DDB2) | (1 << DDB3) | (1 << DDB5));
 #endif
+#ifdef CPU_ATMEGA1284P
```

Comment:
maybe replace the previous by `#if defined(CPU_ATMEGA328P)` and here you can use an `#elif defined(...)`

Reply:
Should we also add an `#else #warning No SPI definition for CPU` afterwards?

Reply:
Sound good, this way we'll not forget to define them! :+1: